web/META-INF/resources/js/controllers.js

```diff
@@ -0,0 +1,266 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you 
+ * may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/* Controllers */
+var mudrodControllers = angular.module('mudrodControllers', []);
+
+mudrodControllers.controller('searchCtrl', ['$scope', '$rootScope', '$location', 'Autocomplete', 'SearchOptions', 
+    function ($scope, $rootScope, $location, Autocomplete, SearchOptions) {
+        $scope.options = {
+            opt: 'And'
+        };
+        //$scope.relatedTerms = [];
+        //$rootScope.searchOptions = {};
+        
+        $scope.complete = function(string) {  
+            $scope.hidethis = false;  
+            $scope.hideoption = false;
+            var output = [];  
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'output' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)